maestro-client/src/main/java/maestro/drivers/AndroidDriver.kt

```diff
@@ -261,6 +261,7 @@ class AndroidDriver(
 
     override fun hideKeyboard() {
         dadb.shell("input keyevent 66")
+        dadb.shell("input keyevent 111")
```

Comment:
Is this because this key is mapped differently 🤔 ?

Reply:
Might be different OS versions or due to different vendors. I was never able to reproduce the first case but another dev reported that code 66 worked for them